docs/stdlib/numbers.rst

```diff
@@ -142,9 +142,10 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: int integer
 
-    A 16-bit signed integer.
+    Represents a 16-bit signed integer.
```

Comment:
I would either revert all of these or replace the "Represents a..." preamble of each with "A type for representing..."